components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml

```diff
@@ -2668,6 +2668,10 @@ components:
           description: Decides whether user consent needs to be skipped during logout flows.
         externalizedConsentPage:
           $ref: '#/components/schemas/ExternalizedConsentPage'
```

Comment:
Is this correct?

Reply:
Yes, this is the previous complex type and it will remove in future.

Reply:
Can't we remove it now itself?